src/Services/Air/templates/AIR_TICKET_REQUEST.handlebars.js

```diff
@@ -16,6 +16,9 @@ module.exports = `
       <air:AirPricingInfoRef Key="{{key}}"/>
       {{/refs}}
       <air:AirTicketingModifiers>
+        {{#refs}}
```

Comment:
I see similar construction on the line 15.
Should the new lines you propose be added, or moved here from there?

Reply:
New lines should be added . If we add child in ticketing then it becomes mandatory to include refs inside AirTicketingModifiers . So both will stay .But we could think of something that will put a condition if any child passenger only then added part will go there.

Reply:
@engrashid it's ok as it is, if it's not breaking anything